support-frontend/assets/pages/subscriptions-redemption/api.js

```diff
@@ -53,7 +53,12 @@ function doValidation(userCode: string, dispatch: Dispatch<Action>) {
 
 function validateUserCode(userCode: string, dispatch: Dispatch<Action>) {
   dispatch({ type: 'SET_USER_CODE', userCode });
-  doValidation(userCode, dispatch);
+  const codeLength = getGlobal('codeLength') || 13;
```

Comment:
how will we know if we are relying on the 13? we will have to also change here if we change the rule, but at least it will be more discoverable via the "find usages" and search for "codeLength" in the JS.
The alternative would be to blow up if it's not present, hopefully before we ship to prod.

Reply:
which reminds me, we should not forget to add post deploy tests for gifting